scripts/package-osx.sh

```diff
@@ -37,14 +37,28 @@ if [ -f scripts/appledev.p12 ] && [ -n "$appledev_p12_password" ]; then
 fi
 
 cd dist
+
+# Create app bundle
 ditto -rsrc --arch x86_64 'MusicBrainz Picard.app' 'MusicBrainz Picard.tmp'
 rm -r 'MusicBrainz Picard.app'
 mv 'MusicBrainz Picard.tmp' 'MusicBrainz Picard.app'
 [ "$CODESIGN" = '1' ] && codesign --keychain $KEYCHAIN_PATH --verify --verbose --deep --sign "$CERTIFICATE_NAME" 'MusicBrainz Picard.app'
+
```

Comment:
```suggestion
Just testing out a suggest
```

Reply:
Interesting. @zas @phw did you know about this?

https://help.github.com/en/articles/commenting-on-a-pull-request#adding-line-comments-to-a-pull-request

See point 6